app/uk/gov/hmrc/tariffclassificationfrontend/views/includes/main.scala.html

```diff
@@ -35,7 +36,7 @@
 
 
 @govuk_wrapper(appConfig = appConfig,
-               title = "Tariff Classification",
+               title = s"${customTitle.getOrElse("Tariff classification")}",
```

Comment:
Why is this s"{}" ? customTitle.getOrElse("Tariff classification")

Reply:
Good point actually - because I had that line as something more complicated before where it needed that, can be simplified now